test/postgresql.test.js

```diff
@@ -263,6 +263,18 @@ describe('postgresql connector', function() {
       });
   });
 
+  it('should support where filter for array type field', function(done) {
+    Post.dataSource.settings.allowExtendedOperators = true;
```

Comment:
After adding this we do not need juggler related PR.
PFB PR Link:
https://github.com/strongloop/loopback-datasource-juggler/pull/1865